src/search/headerBar.jsx

```diff
@@ -9,7 +9,7 @@ import {
 import { useDispatch } from 'react-redux';
 import { useNavigation } from 'react-navigation-hooks';
 import styles from './styles';
-import searchActions from '../redux/search/actionCreator';
+import { searchStockData } from '../redux/search/actionCreator';
```

Comment:
you can use this destructor syntax if you export the methods together, just it was before in the `actionCreator` file